OpenRA.Mods.RA/Traits/LeavesHusk.cs

```diff
@@ -48,11 +76,19 @@ public void Killed(Actor self, AttackInfo e)
 					new ParentActorInit(self),
 					new LocationInit(self.Location),
 					new CenterPositionInit(self.CenterPosition),
-					new OwnerInit(self.Owner),
-					new RaceInit(race),
-					new SkipMakeAnimsInit()
+					new RaceInit(race)
```

Comment:
@penev92 be careful to not confuse the race of the actor with the race of the owner.  They are different things.